### DISCUSSION TITLE
Hot Keys for screen reader users

### DISCUSSION THREAD

**Discussion by BlindRyan** [jamulussoftware/jamulus#2630] [UI Related]
Hi, as a blind computer user I use a screen reader to access the accessible build of Jamulus and so I want to say Thank You for the build, Would it be possible to add a few shrot cut / hot keys to quickly jump to parts of the software for quicker and more efficient navigation, for example when I am in a server jamming with others if I need to mute or solo someone I have to press tab a number of times to get to the person I want to mute or solo, this can be tedious if there are more than  a couple of people in the room perhaps Alt S for Solo and it move you to the first solo check box or Control Alt S to bring up a list of all the solo items on the screen.
Thanks

**Comment by pljones** [comment]
There can be up to 150 channels on a server in up to 8 rows in the UI.  A solution would need to be proposed that was flexible enough to handle this case.  The idea of the pop up list seems like one that could work generally.  There can also be up to eight groups (I think?).  The pop up could offer scrolling to select a specific channel, options to select all or select a group.  Once the selection had been made, the action could be performed.

Would a single pop up with the choice of action against the selected channels work?

**Reply by BlindRyan** [reply]
THi, it might work, the main issue is when I and other blind users are in a room/ server and someone joins we usually need to solo them and if we have many people in with us press tab or shift tab multiple times to find them can be tough so if a pop up list could list the users with their level slider, their mute and their solo items we could use our up and down arrow keys to find them and then press the enter key to mute or solo them
I'm definitly willing to test this out as I am totally blind and use multiple screen readers.
hanks for your reply.

**Comment by ann0see** [comment]
@chigkim is the person who provides the accessible build. Could you please comment?
